fireant/dataset/data_blending.py

```diff
@@ -0,0 +1,108 @@
+from fireant import (
+    Field,
+)
+from fireant.dataset.klass import (
+    DataSet,
+)
+from fireant.queries import (
+    DimensionChoicesQueryBuilder,
+    DimensionLatestQueryBuilder,
+)
+from fireant.queries.builder.dataset_blender_query_builder import DataSetBlenderQueryBuilder
+from fireant.utils import immutable
+
+
+class DataSetBlender(object):
+    """
+    WRITEME
+    """
+
+    def __init__(self, primary_dataset):
+        """
+        Constructor for a data blended slicer.  Contains all the fields to initialize the slicer.
+
+        :param primary_dataset: (Required)
+            The primary dataset, which table will be used as part of the FROM expression.
+        """
+        self.primary_dataset = primary_dataset
+        self.table = primary_dataset.table
```

Comment:
is there a benefit of storing a reference to this separately? What if, for example, someone using this class were to modify the primary dataset value, then this reference would become invalid, right?

Reply:
I've made it a property.